### PR TITLE
Hello world instructors

### PR DESCRIPTION
**Pull request by mikeharv**
Add three new images to be used as "instructor" images in Sprite Lab levels: bear.png, emoji.png, and retro.png

## PR Checklist:

- [n/a ] Tests provide adequate coverage
- [ n/a] Privacy and Security impacts have been assessed
- [n/a ] Code is well-commented
- [X ] New features are translatable or updates will not break translations
- [ n/a] Relevant documentation has been added or updated
- [X ] User impact is well-understood and desirable
- [ X] Pull Request is labeled appropriately
- [n/a ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/db/schema_cache.yml [hunk 1]
```diff
@@ -7130,7 +7130,7 @@ columns:
     comment: 
   lessons_opportunity_standards:
   - &1825 !ruby/object:ActiveRecord::ConnectionAdapters::MySQL::Column
-    name: lesson_id
+    name: id
```
Comment (epeach) [inline dashboard/db/schema_cache.yml]: I think the changes in this file are not relevant to your change, so let's remove any changes to this file from this PR.

**Review by epeach — CHANGES_REQUESTED** [review]
First PR - woo hoo! :tada: I left one comment about a file with unneeded changes. Additionally, it would be great to have more context in the PR description. Here is an example PR when Anjali last added an instructor asset: https://github.com/code-dot-org/code-dot-org/pull/42115. Not a whole lot is needed, but just something that makes it easy to take a look at the description and figure out what the purpose of the PR is (adding new instructor assets).

**Review by epeach — APPROVED** [review]
Yay! Thanks for making those changes - looks good!

**Comment by mikeharv** [comment]
Removed file with unneeded changes. Should just add 3 images for instructor avatars: bear.png, emoji.png, retro.png

bear
emoji
retro

File: dashboard/db/schema_cache.yml (hunk 1 above)
Comment (mikeharv) [reply dashboard/db/schema_cache.yml]: Confirmed as irrelevant. Removed!

**Comment by mikeharv** [comment]
@epeach Do you know why these images aren't showing up here? https://github.com/code-dot-org/code-dot-org/tree/levelbuilder/apps/static/spritelab
I'm still not able to add them to a level. I think I'm not clear on the various directories.

- I added to `code-dot-org/apps/static/spritelab/`.
- I can't seem to find `/tree/levelbuilder/apps/static/spritelab/` locally. Are these different?
- If I open a working avatar from production, it has a URL like https://studio.code.org/blockly/media/spritelab/poster.png

**Pull request merged**